ui/tree/src/ops.ts

```diff
@@ -78,6 +79,7 @@ export function merge(n1: Tree.Node, n2: Tree.Node): void {
       )
         n1.comments.push(c);
     });
+  } else delete n1.comments;
```

Comment:
Same thing here. We want to merge, not replace.

This change would delete user comments when an analysis is requested and the stockfish comment overrides the user comments, instead of being merged to them.